50mariadb.yml

```diff
@@ -37,17 +37,19 @@ spec:
               fieldPath: metadata.namespace
         - name: DATADIR
           value: /data/db
+        - name: AUTO_NEW_CLUSTER
+          value: "false"
```

Comment:
Out of curiousity, what would it mean if I flipped this to true and `kubectl applied`?

Reply:
It'd mean you wouldn't be paged in the middle of the night because we'd be back at `mysql_global_status_wsrep_cluster_size` = 3 fast enough, but possibly with lost writes because we guessed that pod 0 had no replication lag.